app/desktop/log_config.py

```diff
@@ -0,0 +1,103 @@
+import os
+from enum import Enum
+from pathlib import Path
+from typing import List
+
+
+class LogDestination(Enum):
+    CONSOLE = "console"
+    FILE = "file"
+    ALL = "all"
+
+
+# TODO: consolidate this with kiln_server.server.project_api.default_project_path
```

Comment:
Let's use the settings path? More hidden, which limits logs to more technical users 😆 

Already exposed nicely here: 

```
from kiln_ai.utils.config import Config
Config.settings_path()
```
```